bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/system/ITarget.java

```diff
@@ -28,7 +26,7 @@
  */
 public abstract class ITarget implements Cloneable, Serializable {
```

Comment:
@tpolecat Any comments on this? I would like to merge this asap. A next update for the target editor based on this PR is coming soon.